shared/src/main/scala/zeno/examples/MultiPaxosClient.scala

```diff
@@ -0,0 +1,65 @@
+package zeno.examples
+
+import scala.collection.mutable.Buffer
+import scala.concurrent.Future
+import scala.concurrent.Promise
+import scala.scalajs.js.annotation._
+import zeno.Actor
+import zeno.Logger
+import zeno.ProtoSerializer
+import zeno.TypedActorClient
+
+@JSExportAll
+object MultiPaxosClientInboundSerializer
+extends ProtoSerializer[MultiPaxosClientInbound] {
+  type A = MultiPaxosClientInbound
+  override def toBytes(x: A): Array[Byte] = super.toBytes(x)
+  override def fromBytes(bytes: Array[Byte]): A = super.fromBytes(bytes)
+  override def toPrettyString(x: A): String = super.toPrettyString(x)
+}
+
+@JSExportAll
+object MultiPaxosClientActor {
+  val serializer = MultiPaxosClientInboundSerializer
+}
+
+@JSExportAll
+class MultiPaxosClientActor[Transport <: zeno.Transport[Transport]](
+  address: Transport#Address,
+  transport: Transport,
+  logger: Logger,
+  config: PaxosConfig[Transport]
+) extends Actor(address, transport, logger) {
+  override type InboundMessage = MultiPaxosClientInbound
+  override def serializer = MultiPaxosClientActor.serializer
+
+  // The set of replicas.
+  private val proposers
+  : Seq[TypedActorClient[Transport, MultiPaxosReplicaActor[Transport]]] =
+    for (proposerAddress <- config.proposerAddresses)
+      yield
+      typedActorClient[MultiPaxosReplicaActor[Transport]](
+        proposerAddress,
+        MultiPaxosReplicaActor.serializer
+      )
+
+      // valueProposed holds a proposed value, if one has been proposed. Once a
```

Comment:
This was true for Paxos, but I don't think it's true for MultiPaxos. Clients should probably attach client ids to their requests, and they should be allowed to issue multiple requests. Maybe not multiple concurrent requests, but probably at least one request after another.